api.js

```diff
@@ -7,19 +7,26 @@ const api = axios.create({
 const getArticlesByCreateDate = () => {
```

Comment:
do you need this function if the next one has a default sort by param for created at?